### PR TITLE
Dev notion tweets

### PR DESCRIPTION
**Pull request by bajajcodes**
- Add tweet to notion with text and get to know when notion has added tweet to database
- Get notified when useNotion adds data to notion database
- useAxios now reset states on post initial request to add listener effects on event completion such as api call.

### CODE REVIEW

File: src/hooks/useNotion/useNotion.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            },
+          },
+        ],
+      },
+    };
+  }
+
+  function generateTweetBlock(text, media) {
+    const separatedText = text.split("\n");
+    const paragraphs = separatedText.filter(p => p !== "").map((text) =>
```
Comment (deepak29-git) [inline src/hooks/useNotion/useNotion.js:40]: use meaningful name instead of p

**Review by deepak29-git — APPROVED** [review]
rest of looks good to me

**Review by arpit01923 — APPROVED** [review]
- code looks fine, good to merge

File: src/hooks/useNotion/useNotion.js (hunk 1 above)
Comment (bajajcodes) [reply src/hooks/useNotion/useNotion.js:40]: I left this by mistake and from next time I won't be repeating this again.

**Pull request merged**